### PR TITLE
[IOTDB-443] Fix ReadOnlyMemChunk round float and double data incorrectly

### PR DESCRIPTION
**Pull request by samperson1997**
Fix ReadOnlyMemChunk round float and double data incorrectly for Gorilla encoding.

### CODE REVIEW

**Comment by liutaohua** [comment]
maybe this pr need to merge into new_series_reader

**Comment by samperson1997** [comment]
> maybe this pr need to merge into new_series_reader

Right. I'll change the merge request.

File: server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java [hunk 1]
```diff
@@ -86,23 +86,14 @@
...
- * When a TsFileProcessor is closed, the closeUnsealedTsFileProcessor() method will be called as a
- * callback.
+ * TsFileProcessor in the working status. <br/> <p> There are two situations to set the working
+ * TsFileProcessor to closing status:<br/> <p> (1) when inserting data into the TsFileProcessor, and
+ * the TsFileProcessor shouldFlush() (or shouldClose())<br/> <p> (2) someone calls
+ * waitForAllCurrentTsFileProcessorsClosed(). (up to now, only flush command from cli will call this
+ * method)<br/> <p> UnSequence data has the similar process as above. <p> When a sequence
+ * TsFileProcessor is submitted to be flushed, the updateLatestFlushTimeCallback() method will be
+ * called as a callback.<br/> <p> When a TsFileProcessor is closed, the
+ * closeUnsealedTsFileProcessor() method will be called as a callback.
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java]: Hi, do not FORMAT this paragraph...

Comment (samperson1997) [reply server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java]: Sorry my fault... I have reversed all formats like this.

**Pull request merged**